data_steward/tools/create_ehr_and_rdr_snapshot.sh

```diff
@@ -26,18 +36,18 @@ while true; do
   esac
 done
 
-if [[ -z "${key_file}" ]] || [[ -z "${ehr_dataset}" ]]; then
-  echo "Specify the key file location and ehr_dataset ID. $USAGE"
+if [[ -z "${key_file}" ]] || [[ -z "${ehr_dataset}" ]] || [[ -z "${rdr_dataset}" ]] || [[ -z "${dataset_release_tag}" ]]; then
+  echo "Specify the key file location and ehr_dataset ID, rdr_dataset ID and Dataset release tag . $USAGE"
   exit 1
 fi
 
-today=$(date '+%Y%m%d')
-app_id=$(python -c 'import json,sys;obj=json.load(sys.stdin);print(obj["project_id"]);' < "${key_file}")
+app_id=$(python -c 'import json,sys;obj=json.load(sys.stdin);print(obj["project_id"]);' <"${key_file}")
```

Comment:
Did you intentionally remove the space between `<"`?

Reply:
it is wrong formatting i'll update it.